test_plugin/src/lib.rs

```diff
@@ -88,3 +102,27 @@ fn op_test_resource_table_close(
   let result_box: Box<[u8]> = Box::new(*result);
   Op::Sync(result_box)
 }
+
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct JsonOpArgs {
+  name: String,
+}
+
+pub fn op_test_json_op_sync(
+  _state: &mut OpState,
+  args: JsonOpArgs,
+  _bufs: &mut [ZeroCopyBuf],
+) -> Result<Value, AnyError> {
+  println!("Hello {} from sync json op!", args.name);
+  Ok(json!(format!("Hello {}!", args.name)))
+}
+
+pub async fn op_test_json_op_async(
+  _state: Rc<RefCell<OpState>>,
+  args: JsonOpArgs,
+  _bufs: BufVec,
+) -> Result<Value, AnyError> {
+  println!("Hello {} from async json op!", args.name);
+  Ok(json!(format!("Hello {}!", args.name)))
```

Comment:
Would make sense to do some async tasks inside this op?

Reply:
Makes a lot of sense :P